de

```diff
@@ -18,9 +18,15 @@ function fail {
     fi
     exit 1
 }
+function storeInput { echo "$1"; echo -n "${2}=" >> ~/.de; read ${2}; echo "${!2}" >> ~/.de; }
 
+task "Running config file"
```

Comment:
This is something that would happen every time. I don't think we need to tell this to the user each run.